bcgov/onroutebc
# ORV2-761: Associated error correlation id with error handling screens

# Description

This PR created the logic to associate an error correlation ID with error handling screens. An error correlation ID will be generated for a specific error and log to database for error tracking. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Test steps:

1. Login to frontend with bceid test account.
2. Navigate to <root_rul>/111 for testing NotFound errors.
3. Expecting an error not found page showing and verify if a error record can be created in the backend database with associated session id and correlation id.
4. There are two tables are new created for the errors, dbo.ORBC_ERROR and dbo.ORBC_VT_ERROR_TYPE.

Jira ticket: 
https://jira.th.gov.bc.ca/browse/ORV2-761

- Tested locally

## Checklist






## Further comments




---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend - Vehicles](https://onroutebc-598-backend-vehicles.apps.silver.devops.gov.bc.ca/) available
[Backend - DOPS](https://onroutebc-598-backend-dops.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://onroutebc-598-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge-main.yml)
